src/js/themes/base.js

```diff
@@ -105,6 +105,22 @@ export const generate = (baseSpacing = 24, scale = 6) => {
 
   const result = deepMerge(iconBase, {
     global: {
+      active: {
+        background: {
+          dark: {
```

Comment:
Since they are identical, can we remove the light and dark keys?

Reply:
hm shouldn't we leave them so that people can configure it if they want it differently?

Reply:
Hmm... That sounds like leaving the extra code just for documentation reasons. Since you could simplify it in a custom theme to not have dark/light separated, it seems like we should document both varieties, which is true for just about all of our color usage. I was proposing keeping the code as functionally clean as possible and leaving the theme documentation text to describe the various options further.

Reply:
oh right ok! i thought removing the functionnality! +1